apps/telegram-bot/core/database.py

```
@@ -0,0 +1,37 @@
+async def startup() -> None:
+    client = AsyncIOMotorClient(MONGODB_URL)
+    await init_beanie(database=client["telegram_bot"], document_models=document_list)
+
+
+async def shutdown() -> None:
+    pass
+
+
+async def save_instances(instances: Union[Document, List[Document]], *args) -> None:
+    if instances is None:
+        raise TypeError("instances must be a Model or a list of Model")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)